app/src/test/java/me/hufman/androidautoidrive/TestMusicApp.kt

```diff
@@ -1265,28 +1424,67 @@ class TestMusicApp {
 				MusicMetadata("bonusFolder1", title = "BonusFolder1", browseable = true, playable = false),
 				browseView.pageStack[1].previouslySelected)
 
+		// select a deeper folder (bonusFolder2:BonusFolder2, to show in BROWSE2)
```

Comment:
Good, make sure the wraparound test is still present

Reply:
Yea I have it so now it's browsing a level deeper which means it will hit that wraparound case BROWSE1 -> BROWSE2 -> BROWSE3 -> BROWSE2, I spent way too much time on getting that test to work all because I messed up a single focus event 😆